### PR TITLE
HDDS-4829. Rename MiniOzoneHACluster to MiniOzoneOMHACluster.

### PR DESCRIPTION
**Pull request by mukul1987**
## What changes were proposed in this pull request?
Rename MiniOzoneHACluster to MiniOzoneOMHACluster

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-4829

## How was this patch tested?
Simple refactoring change.

### CODE REVIEW

File: hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/MiniOzoneOMHAClusterImpl.java [hunk 1]
```diff
@@ -54,10 +54,10 @@
  * with OM HA suitable for running tests.  The cluster consists of a set of
  * OzoneManagers, StorageContainerManager and multiple DataNodes.
  */
-public class MiniOzoneHAClusterImpl extends MiniOzoneClusterImpl {
```
Comment (bharatviswa504) [inline hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/MiniOzoneOMHAClusterImpl.java:57]: Comments need to be updated witth MiniOzoneHAClusterImpl -> MiniOzoneOMHAClusterImpl

File: hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestMiniOzoneOMHACluster.java [hunk 2]
```diff
@@ -41,9 +41,9 @@
 /**
  * This class tests MiniOzoneHAClusterImpl.
```
Comment (bharatviswa504) [inline hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestMiniOzoneOMHACluster.java:42]: same here comments need to be updated

File: hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/recon/TestReconWithOzoneManagerHA.java [hunk 3]
```diff
@@ -55,7 +55,7 @@
   @Rule
   public Timeout timeout = new Timeout(300_000);
 
-  private MiniOzoneHAClusterImpl cluster;
+  private MiniOzoneOMHAClusterImpl cluster;
```
Comment (bharatviswa504) [inline hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/recon/TestReconWithOzoneManagerHA.java:58]: Same here comments need to be updated

**Review by bharatviswa504 — COMMENTED** [review]
Overall LGTM.
Few minor comments.

**Comment by amaliujia** [comment]
+1 this is a good step for supporting MiniOzoneSCMHACluster

**Comment by amaliujia** [comment]
Question: 
Do you need to overwrite  `waitForClusterToBeReady` in `MiniOzoneOMHACluster` (e.g. to wait OM elects a leader)?

**Comment by bshashikant** [comment]
> Question:
> Do you need to overwrite `waitForClusterToBeReady` in `MiniOzoneOMHACluster` (e.g. to wait OM elects a leader)?
Leader election should be handled within safe mode itself.

**Comment by bshashikant** [comment]
> > Question:
> > Do you need to overwrite `waitForClusterToBeReady` in `MiniOzoneOMHACluster` (e.g. to wait OM elects a leader)?
> > Leader election should be handled within safe mode itself.

**Review by bharatviswa504 — APPROVED** [review]
+1 LGTM

**Review by bshashikant — APPROVED** [review]

**Pull request merged**